tensorflow/lite/kernels/test_util.h

```diff
@@ -15,6 +15,8 @@ limitations under the License.
 #ifndef TENSORFLOW_LITE_KERNELS_TEST_UTIL_H_
 #define TENSORFLOW_LITE_KERNELS_TEST_UTIL_H_
 
+#include <gmock/gmock.h>
+#include <gtest/gtest.h>
```

Comment:
These lines should go back to where they were.

Reply:
Addressed the requested change in [#1b42f81](https://github.com/MCW-Dev/tensorflow/pull/31/commits/1b42f812cab737e1f4afff6d89506aa5c1aeb0bc)